src/components/Session/HistoryTable.jsx

```diff
@@ -0,0 +1,77 @@
+/*
+  This file contains the Effects History Table Component.
+  This component displays relevant information about
+  all the effects of an account in descending chronological order.
+  It is meant to be visual, concise, scanable, minimal, and
+  clean, with four catagories:
+  1) Effect type
+  2) Sub action (ie CREATE for account_create)
+  3) Description
+  4) Date (time since and ledger sequence number)
+*/
+const React = window.React = require('react');
+import EffectCard from './EffectCard.jsx';
+import DateCard from './DateCard.jsx';
+import {daysSince} from '../../lib/Format';
+
+export default class HistoryTable extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    let account = this.props.d.session.account;
+
+    // From MagicSpoon.Account
```

Comment:
State should be in the "Driver" somewhere: https://github.com/irisli/stellarterm/blob/master/src/lib/Driver.js